### PR TITLE
Documentation, dependency install command (macOS, Homebrew)

### PR DESCRIPTION
**Pull request by justingeeslin**
Including the command to install dependency gfortran for macOS via Homebrew. Hopefully this will be helpful for a first-timer.

### CODE REVIEW

**Review by Keno — APPROVED** [review]
Seems good to me.

File: doc/build/macos.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
 ## macOS
 
 You need to have the current Xcode command line utilities installed: run `xcode-select --install` in the terminal.
+
 You will need to rerun this terminal command after each macOS update, otherwise you may run into errors involving missing libraries or headers.
+
 You will also need a 64-bit gfortran to compile Julia dependencies. The gfortran-4.7 (and newer) compilers in Homebrew work for building Julia.
+```bash
```
Comment (fredrikekre) [inline doc/build/macos.md]:
```suggestion
```bash
```

**Comment by ViralBShah** [comment]
@staticfloat Isn't gfortran part of the gcc install on brew nowadays?

**Comment by ViralBShah** [comment]
Also, given BinaryBuilder, I think we no longer need `gfortran` to build julia. Perhaps this PR should be updated accordingly.

File: doc/build/macos.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
 ## macOS
 
 You need to have the current Xcode command line utilities installed: run `xcode-select --install` in the terminal.
+
 You will need to rerun this terminal command after each macOS update, otherwise you may run into errors involving missing libraries or headers.
+
 You will also need a 64-bit gfortran to compile Julia dependencies. The gfortran-4.7 (and newer) compilers in Homebrew work for building Julia.
+```bash 
+brew cask install gfortran
```
Comment (staticfloat) [inline doc/build/macos.md]: You should instead just `brew install gcc`, rather than installing a `gfortran` cask.

**Review by staticfloat — CHANGES_REQUESTED** [review]

**Pull request merged**